server.js

```diff
@@ -1,38 +1,34 @@
 // TASK 1: Setting up server
 
 
-
```

Comment:
The HOSTNAME variable name is in uppercase which is not standard. It should be lowercase.